Gemfile.rails42

```diff
@@ -0,0 +1,12 @@
+source 'https://rubygems.org'
+gemspec
+
+gem 'jruby-openssl', :platforms => :jruby
+
+group :test, :remote_test do
+  # gateway-specific dependencies, keeping these gems out of the gemspec
+  gem 'braintree', '>= 2.0.0'
+  gem 'oauth', '0.4.7'
+end
```

Comment:
Cool, I'll leave it like this.

I'd like to get rid of the braintree gem, and maybe standadize on a shared oauth gem that is not shitty. But not now.